applications/nrf_desktop/doc/hid_forward.rst

```diff
@@ -76,13 +78,18 @@ Forwarding HID input reports
 ============================
 
 After :ref:`nrf_desktop_ble_discovery` successfully discovers a connected peripheral, the |hid_forward| automatically subscribes for every HID input report provided by the peripheral.
+Reports in both HID report protocol and HID boot protocol can be used by the subscriber.
```

Comment:
Subscriber can use either HID boot or HID report protocol, both protocols cannot be used at the same time.